### ISSUE TITLE
ARIA - Screen reader does not announce that user in a text field inside of combobox

### ISSUE THREAD

**Issue by Confidenceman02** [Confidenceman02/elm-select#5]
When a user keyboard focuses into the select the voice over and screen reader does not announce that the element is a combobox. The reason this happens is because keyboard focusing does not open the menu and thus does not render a <ul> element with the role of listbox.

Conversely, click focusing onto the select opens the menu and correctly render a <ul> with role=listbox. The screen reader correctly announces that the user in is a text field inside of a combobox.

## Proposed solution
By default even with the menu closed, if the select is focused we should be rendering a <ul> with role=listbox for the screen reader to correctly announce the element.
